build/rpm/custom-strip

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "STRIPPING IN $*"
+
```

Comment:
This script could do with a little comment describing why it exists.

Reply:
fixed in cb8b03215

Reply:
fixed in cb8b03215b1be9fd45dfde55ba6720a2d3969e9f